services/aws-s3.service.ts

```diff
@@ -1,10 +1,17 @@
 import angular from "angular";
-import AWS from "aws-sdk";
+import {
+  S3Client,
+  GetObjectCommand,
+  GetObjectCommandInput,
+} from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
 import { AppCredentialsService } from "./credentials.service";
 
 export class AppAwsS3Service {
   readonly bucketRegex = /http.?:\/\/([-\w]+).*.com/;
   readonly keyRegex = /s3.amazonaws.com\/(.*)/;
+  readonly defaultRegion = "us-west-2";
```

Comment:
spoke offline, but otr-service and otr-service-devo buckets are on us-west-2 so makes to default here